libs/stanag4609/src/main/java/org/codice/alliance/libs/stanag4609/Stanag4609TransportStreamParser.java

```diff
@@ -385,7 +386,7 @@
 
     securityLocalSetContext.addDataElement(
         new KlvUnsignedByte(new byte[] {12}, OBJECT_COUNTRY_CODING_METHOD));
-    securityLocalSetContext.addDataElement(new KlvString(new byte[] {13}, OBJECT_COUNTRY_CODES));
+    securityLocalSetContext.addDataElement(new KlvBytes(new byte[] {13}, OBJECT_COUNTRY_CODES));
```

Comment:
Just curious. What is the difference between `KlvString` and `KlvBytes`?

Reply:
KlvString can only return a string where KlvBytes is the raw bytes